### ISSUE TITLE
intervalls of hours, minutes.

### ISSUE THREAD

**Issue by transfluxus** [commodityvectors/d3-timeline#25]
hi,
is there a possibility to have hour or minute intervalls. it doesn't work atm.
 they can't be displayed... :/

**Comment by wumingzhang** [comment]
agree.. if possible to do something like new Date([2015, 1, 15, 12, 30]) for 2015-01-15 12:30 GMT could be cool?
